localstack/ext/java/src/main/java/cloud/localstack/LocalstackTestRunner.java

```diff
@@ -192,11 +237,18 @@ public static boolean isEnvConfigSet(String configName) {
 	}
 
 	private static String getEndpoint(String service) {
+		LOG.info(String.format("fetching endpoint for '%s'...", service));
```

Comment:
Perhaps we can make this a `debug` message instead of `info`? (same for line 242)